src/server/server.js

```diff
@@ -68,8 +68,8 @@ app.get('/football/teams/:teamid/players', (req, res) => {
   searchFootballTeamPlayers(res, req.params.teamid);
 });
 
-app.get('/journalism/teams/:teamname', (req, res) => {
-  journalismTeam(res, req.params.teamname);
+app.get('/journalism/teams/:teamname/:footballDataOrgTeamId', (req, res) => {
+  journalismTeam(res, req.params.teamname, req.params.footballDataOrgTeamId);
```

Comment:
Agreed, basically decided to do it this way because it's easier and to be fair, with the football data api, a football team name could have more than one id depending on what year it's from so I justified it that way :P